plugins/source/aws/docs/tables/aws_lightsail_database_log_events.md

```diff
@@ -14,11 +14,11 @@ This table depends on [aws_lightsail_databases](aws_lightsail_databases.md).
 
 | Name          | Type          |
 | ------------- | ------------- |
-|_cq_id (PK)|`uuid`|
+|_cq_id|`uuid`|
 |_cq_parent_id|`uuid`|
 |account_id|`utf8`|
 |region|`utf8`|
-|database_arn|`utf8`|
-|created_at|`timestamp[us, tz=UTC]`|
+|database_arn (PK)|`utf8`|
+|created_at (PK)|`timestamp[us, tz=UTC]`|
```

Comment:
I don't think timestamp is unique enough

Reply:
I was inspired by [`aws cli lightsail get-relational-database-events`](https://docs.aws.amazon.com/cli/latest/reference/lightsail/get-relational-database-events.html) doc here.

I presume, there may be a clashing event for a DB, but IMO it's highly unlikely for now.

We can add the concatenated categories to the PK, though. WDYT?